packages/ocaml-freestanding/ocaml-freestanding.0.3.1/opam

```diff
@@ -0,0 +1,25 @@
+opam-version: "1.2"
+maintainer: "Martin Lucina <martin@lucina.net>"
+authors: "Martin Lucina <martin@lucina.net>"
+homepage: "https://github.com/mirage/ocaml-freestanding"
+bug-reports: "https://github.com/mirage/ocaml-freestanding/issues/"
+license: "MIT"
+tags: "org:mirage"
+dev-repo: "https://github.com/mirage/ocaml-freestanding.git"
+build: [make]
+install: [make "install" "PREFIX=%{prefix}%"]
+remove: [make "uninstall" "PREFIX=%{prefix}%"]
+depends: [
+  "conf-pkg-config"
+  "ocamlfind"
```

Comment:
Are you using the `Findlib` library? If not you might want to flag this as a `{build}` dependency.
Same with the one above I think.

Reply:
@kit-ty-kate I only use the `ocamlfind` command at build time, so yes, this can be a build dependency. In fact, same for `ocaml-src`. `conf-pkg-config` has to stay as-is since `pkg-config` (which it installs via depext) is required at "run" time to build unikernels. Will push a fix.